recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: nibabel
+  version: "0.11"
+
+source:
+    git_url: https://github.com/nipy/nibabel.git
+    git_tag: 2.0.2
+
+requirements:
+  build:
+    - python
+    - numpy
+  run:
+    - python
+    - numpy
```

Comment:
I believe it does require numpy: https://github.com/nipy/nibabel/blob/master/requirements.txt